saleor/graphql/shipping/types.py

```diff
@@ -248,7 +249,7 @@ def resolve_channels(root: ChannelContext[models.ShippingZone], info, **_kwargs)
         return ChannelsByShippingZoneIdLoader(info.context).load(root.node.id)
 
 
-class ShippingMethod(ChannelContextType, MetadataMixin):
+class ShippingMethod(ObjectType):
```

Comment:
Maybe I am missing something, but you dropped a MetadataMixin, which means that you will not have a metadata fields here. It is strange as I don't see an any scheme change 🤔

Reply:
Metadata fields are being added by the interface defined down below. MetadataMixin was there to unwrap the ChannelContext which is now obsolete